changelogs/unreleased/bump-metrics-to-1.3.0.md

```diff
@@ -0,0 +1,6 @@
+## feature/lua
+
+* Added new section to `metrics`: `cpu_extended`. The section includes
+  `tnt_cpu_thread`, `tnt_cpu_number` and `tnt_cpu_time` metrics for instances
+  running on Linux. The section is enabled by default.
+* Fixed using box.cfg.read_only instead of box.info.ro in replication metrics.
```

Comment:
```suggestion
* Added a new section to `metrics`: `cpu_extended`. The section includes
  `tnt_cpu_thread`, `tnt_cpu_number` and `tnt_cpu_time` metrics for instances
  running on Linux. The section is enabled by default.
* Fixed using `box.cfg.read_only` instead of `box.info.ro` in replication metrics.
```

Reply:
Updated based on https://github.com/tarantool/tarantool/pull/11243#discussion_r1992338031